src/components/RubberStampToolButton/RubberStampToolButton.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+
+import Button from 'components/Button';
+
+import core from 'core';
+import actions from 'actions';
+import selectors from 'selectors';
+
+const RubberStampToolButton = () => {
+  const [isSignatureOverlayOpen] = useSelector(
+    state => [
+      selectors.isElementOpen(state, 'stampOverlay'),
+    ],
+    shallowEqual,
+  );
+  const dispatch = useDispatch();
+
+
+  const handleClick = () => {
+    dispatch(actions.toggleElement('stampOverlay'));
+  };
+
+  const buttonClass = classNames({
+    'down-arrow': true,
+  });
+
+  return (
+    <Button
+      className={buttonClass}
+      isActive={isSignatureOverlayOpen}
+      img="ic_annotation_stamp_black_24px"
+      onClick={handleClick}
+      title="annotation.rubberStamp"
```

Comment:
I noticed that the tooltip of this button doesn't contain a hotkey. I think we should add a hotkey for this button to match other buttons in the header. Do you think this makes sense?